playbooks/openshift-glusterfs/private/upgrade.yml

```diff
@@ -1,8 +1,36 @@
 ---
+- name: GlusterFS Upgrade Checkpoint Start
+  hosts: all
```

Comment:
```suggestion
  hosts: OSEv3
```

Reply:
I'm just following the patterns I'm seeing in other files. Is this a new standard?

Reply:
This has always been a standard Gluster has been ignoring :)

The inventory may contain hosts, which are not part of OSE, so we should use OSEv3 group to minimize changes

Reply:
Gotcha.